.golangci.yml

```diff
@@ -37,6 +38,87 @@ linters:
         - name: superfluous-else
         - name: var-declaration
         - name: duplicated-imports
+    tagliatelle:
+      case:
+          rules:
+            json: goCamel
+            yaml: goCamel
+          ignored-fields:
```

Comment:
for each field, could you add as a comment the actual tag name vs the expected one, for reference and to help with review?

Reply:
I don't see that change